packages/cli/src/runner/crawler-command-runner.ts

```diff
@@ -25,5 +39,34 @@ export class CrawlerCommandRunner implements CommandRunner {
             existingUrls: scanArguments.existingUrls,
             discoveryPatterns: scanArguments.discoveryPatterns,
         });
+
+        const endDate = new Date();
+        const endDateNumber = Date.now();
+        const durationSeconds = (endDateNumber - startDatNumber) / 1000;
+        console.log(`Done in ${durationSeconds}s`);
+
+        const scannedPagesCount = scanResult.summaryScanResults.failed.length + scanResult.summaryScanResults.passed.length;
+        const discoveredPagesCount = scannedPagesCount + scanResult.summaryScanResults.unscannable.length;
+        console.log(`Scanned ${scannedPagesCount} of ${discoveredPagesCount} pages discovered `);
+
+        const issueCount = scanResult.summaryScanResults.failed.reduce((a, b) => a + b.numFailures, 0);
+        console.log(`Found ${issueCount} accessibility issues`);
+
+        const crawlDetails: CrawlSummaryDetails = {
+            baseUrl: scanArguments.url,
+            basePageTitle: '',
+            scanStart: startDate,
+            scanComplete: endDate,
+            durationSeconds: durationSeconds,
+        };
+
+        const reportContent = await this.reportGenerator.generateSummaryReport(crawlDetails, scanResult.summaryScanResults);
+
+        const reportLocation = this.reportDiskWriter.writeToDirectory(scanArguments.output, 'index', 'html', reportContent);
+        console.log(`Summary report was saved as ${scanArguments.output}\\${reportLocation}`);
+
+        const errorLogLocation = `${this.reportNameGenerator.generateName('ai-cli-errors', endDate)}.log`;
+        this.reportDiskWriter.writeErrorLogToDirectory(scanArguments.output, errorLogLocation, scanResult.errors);
+        console.log(`Error log was saved as ${scanArguments.output}\\${reportLocation}`);
```

Comment:
will not work on linux. Use filenamify to convert path

Reply:
I'll use 'path' package